jac-uk/apply
# Bugfix/TS 496 Self assessment extraction word counts

## What's included?

Issue: [User Raised Issue BR_ADMIN_PR_000274 #496](https://github.com/jac-uk/ticketing-system/issues/496)

- Upgrade the jac-kit version to `4.1.53`.
- Use the function `splitWords` from jac-kit in the `FormField` component.

## Who should test?
✅ Product owner
✅ Developers

## How to test?
[Example vacancy](https://jac-apply-develop--pr1275-bugfix-ts-496-self-a-whdn4fy1.web.app/vacancy/gWHwfBAlA9JYqzhwELnx/)

Steps:
1. Apply for the vacancy and go to the "Self assessment with competencies" section.
2. Upload a finished self assessment and check if the word count is correct.

    For example, the word count for `It’s something about` and `It's something about` should be 3.


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/user-attachments/assets/fd83dc35-62fd-4bf2-a850-05bcf3a3e8a4

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
